src/components/App.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import Page from './Page';
```

Comment:
```suggestion
import React, { useState } from 'react';

import Page from './Page';
```

의미상으로 구분되는 경우 빈 줄을 넣어 구분해주시면 좋아요.
빈 줄 코드에도 하나하나 의미를 담아서 작성하신다고 생각하시면 좋아요 ;)

Reply:
넵 유의하겠습니다!